packages/syft/src/syft/core/node/common/client_manager/domain_api.py

```diff
@@ -41,49 +40,41 @@ def all(self, pandas: bool = True) -> List[Any]:
         result = response.payload.kwargs  # type: ignore
 
         if result["status"] == "ok":
-            _data = result["data"]
-            if (
-                self.cache is None
-                or (time.time() - self.cache_time > self.timeout)
-                or len(_data) != self.num_known_domains_even_offline_ones
-            ):
-                # check for logged in domains if the number of possible domains changes (if a new domain shows up)
-                self.num_known_domains_even_offline_ones = len(_data)
-                n = len(_data)
-                data = list()
-
-                args = [
-                    (self, i, n, domain_metadata["id"])
-                    for i, domain_metadata in enumerate(_data)
-                ]
-
-                # Check domain status in parallel
-                # check_status = parallel_execution(check_domain_status)
-                # result = check_status(args=args) if args else []
-
-                # for i, status in enumerate(result):
-                #     if status is True:
-                #         data.append(_data[i])
-
-                # Check domain status sequentially
-                for i, arg in enumerate(args):
-                    if check_domain_status(*arg):
-                        data.append(_data[i])
-
-                sys.stdout.write("\r                                             ")
-
-                self.cache = data
-                self.cache_time = time.time()
-            else:
-                data = self.cache
+            data = result["data"]
+            # if (
+            #     self.cache is None
+            #     or (time.time() - self.cache_time > self.timeout)
+            #     or len(_data) != self.num_known_domains_even_offline_ones
+            # ):
+            #     # check for logged in domains if the number of possible domains changes (if a new domain shows up)
+            #     self.num_known_domains_even_offline_ones = len(_data)
+            #     # n = len(_data)
+            #     # data = list()
+            #     # args = [
+            #     #     (self, i, n, domain_metadata["id"])
+            #     #     for i, domain_metadata in enumerate(_data)
+            #     # ]
+
+            #     # # Check domain status sequentially
+            #     # for i, arg in enumerate(args):
+            #     #     if check_domain_status(*arg):
```

Comment:
@madhavajay should we re-enable this, since all messages are sent with timeouts or at least have some sort of default timeout?